src/main.ts

```diff
@@ -94,13 +94,15 @@ async function bootstrap() {
   server.setDefaultRoute((req, res) => {
     if (req.url && req.url.startsWith('/api')) {
       res.statusCode = 404;
-      return res.end({
-        success: false,
-        error: {
-          kind: 'user_input',
-          message: 'Not Found',
-        },
-      });
+      return res.end(
+        JSON.stringify({
```

Comment:
though defined as any in legacy types - expect string, Buffer, or Uint8Array , and not an object